packages/modal/package.json

```diff
@@ -41,6 +41,6 @@
     "@crave/farmblocks-theme": "^1.6.2",
     "lodash.omit": "^4.5.0",
     "lodash.pick": "^4.4.0",
-    "react-spring": "^8.0.20"
+    "react-spring": "^8.0.27"
```

Comment:
What do you guys think of moving this to `peerDependencies`? Since it's used in webapp as well.

cc @MateusStanki

Reply:
I'm ok with that

Reply:
Just add this information to modal docs, please!